<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceClass</class>
 <widget class="QWidget" name="InterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>120</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Conversor &amp; Avaliador</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayoutExpression">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="labelValidation">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Digite a express√£o desejada:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditExpression">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonConvert">
       <property name="text">
        <string>Converter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEvaluate">
       <property name="text">
        <string>Avaliar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="labelResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Resultado:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditResult">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
